FreshRSS/FreshRSS
# Fix: nord theme mobile

Fixed some Nord theme issues:

Before:
![grafik](https://user-images.githubusercontent.com/1645099/203612954-b9cdd678-2cad-4461-a564-58cf4fb45db4.png)


After:
1) background = dark. So it is possible to read the light font color better
2+3) padding improved for mobile view
4) label text is on the left side in mobile view
![grafik](https://user-images.githubusercontent.com/1645099/203612586-00b947e8-8dc8-4131-891d-abeabe0cd009.png)

Not in screenshot:
fixed: position of favicon in normal view in mobile view

Changes proposed in this pull request:

- CSS for Nord theme
- centralized the left/right label text alignment for mobile/desktop view for all themes into frss.css

How to test the feature manually:

1. use Nord theme on small screen (mobile view)
2. check the other themes on a small screen too
3. check it on the display settings page

Pull request checklist:
